### DISCUSSION TITLE
Cannot add a custom domain: "Domain is already owned by another Vercel account."

### DISCUSSION THREAD

**Discussion by tsathis** [vercel/community#89] [Help] [answered]
Referring https://github.com/is-a-dev/register/issues/249

[is-a-dev](https://github.com/is-a-dev) provides `xxx.is-a.dev` domains. I was trying to use tharindu.is-a.dev to a Vercel app which is deployed in http://tharindu.vercel.app, but get the following error, that "domain tharindu.is-a.dev is already owned by another Vercel account.".

Why it says that domain is already owned by someone? How can I use that domain?

![image](https://user-images.githubusercontent.com/34299693/95980801-4e08f000-0e3b-11eb-96fb-962e92f2ff53.png)

**Comment by mcsdevv** [comment, marked as answer]
Hello, we've replied to the ticket you created for this, that'll be the best place for us to help you on this.

**Reply by myqtab** [reply]
this is not helpful, answer so that others can learn

**Reply by utkallabs** [reply]
what's the problem if others get to know this?

**Reply by phenax** [reply]
@mcsdevv, can you provide some more information about this issue in relation to sub-domains?

**Reply by mcsdevv** [reply]
Hi there! As mentioned by the error, the domain is in use on another account.

This can happen in various different ways but is commonly because of account deletions where the domain remains present on the account for 4 days to allow for a full recovery if required.

If anyone requires this process to be expedited, they need to contact Vercel Support.

**Reply by phenax** [reply]
I was asking about subdomains. The issue was created with a subdomain that was never used on vercel. A different sub-domain may have been used in the past by a different user but I'm not sure why the restriction would apply to 2 different sub domains.

**Reply by mcsdevv** [reply]
Hi @phenax, I suggest [contacting support](https://vercel.com/support/request) so that we can investigate that further for you - the behavior you're describing would not be correct.

**Comment by onedebos** [comment]
Leaving this link from the vercel documentation here incase anyone runs into this issue in the future.

https://vercel.com/support/articles/using-domain-after-deleting-account

**Reply by gustavocd** [reply]
The link is no longer valid 😔

**Comment by anondev-sudo** [comment]
I have the same problem, my friend owns test1.domain.com and I can't add test2.domain.com as domain for my project. Please allow that as not everyone have money to get own domain, someone may want to use subdomain from friend for vercel project

**Comment by nassimessaidi** [comment]
I'm already run into this issue and open a ticket for that, the problem is that I linked one of my subdomains on Hashnode, that's why this error happens to me, But I fixed it Now.

If you want to know more if you have a different case, I wrote about that in my blog:

[Domain Is Already Owned By Another Vercel Account Solution](https://www.essaidi.me/Domain-is-already-owned-by-another-Vercel-account-Solution)

**Comment by Pieparker** [comment]
@gustavocd et al, we appreciate you reaching out! Unfortunately, this topic is rather old and has been marked as Answered. If you’re exhibiting the same or a similar issue to the original poster and you're unable to find the help you need, please feel free to create your own discussion and provide as much context as you can. This helps us to ensure that discussions are relevant, fresh and free of any possibly misleading context.
